index.html

```diff
@@ -95,134 +107,136 @@ <h1 class="text-recent-work">My Recent Works</h1>
                     </div>
-                    <div class="stack-single-work-container">
-                        <ul class="stack-single-container">
-                            <li class="stack"><a href="#">Ruby on rail</a></li>
-                            <li class="stack"><a href="#">css</a></li>
-                            <li class="stack"><a href="#">Javascript</a></li>
-                            <li class="stack"><a href="#">html</a></li>
-                        </ul>
-                    </div>
-                    <div class="see-project-container">
-                        <button class="see-project-text btn" type="button">See Project</button>
+                    <div class="infos-container">
+                        <div class="name-single-work">
+                            <h2>Multi-Post Stories Gain+Glory</h2>
+                        </div>
+                        <div class="stack-single-work-container">
+                            <ul class="stack-single-container">
+                                <li class="stack"><a href="#">Ruby on rail</a></li>
+                                <li class="stack"><a href="#">css</a></li>
+                                <li class="stack"><a href="#">Javascript</a></li>
+                                <li class="stack"><a href="#">html</a></li>
+                            </ul>
+                        </div>
+                        <div class="see-project-container">
+                            <button class="see-project-text btn" type="button">See Project</button>
+                        </div>
                     </div>
                 </div>
             </div>
-        </div>
-        <div class="single-card-container">
-            <div class="single-card">
-                <div class="img-single-work">
-                </div>
-                <div class="infos-container">
-                    <div class="name-single-work">
-                        <h2>Multi-Post Stories Gain+Glory</h2>
-                    </div>
-                    <div class="stack-single-work-container">
-                        <ul class="stack-single-container">
-                            <li class="stack"><a href="#">Ruby on rail</a></li>
-                            <li class="stack"><a href="#">css</a></li>
-                            <li class="stack"><a href="#">Javascript</a></li>
-                            <li class="stack"><a href="#">html</a></li>
-                        </ul>
+            <div class="single-card-container">
+                <div class="single-card">
+                    <div class="img-single-work">
                     </div>
-                    <div class="see-project-container">
-                        <button class="see-project-text btn" type="button">See Project</button>
+                    <div class="infos-container">
+                        <div class="name-single-work">
+                            <h2>Multi-Post Stories Gain+Glory</h2>
```

Comment:
- [ ] [OPTIONAL] Good job so far! If you change the `color` to the original(`#172B4D`) before the merge, it would be better 😃 

![image](https://user-images.githubusercontent.com/63915024/206532024-de7b1619-5d3f-4f64-8ac9-05d5c605441f.png)